x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -108,121 +98,6 @@ describe('Resolver Data Middleware', () => {
 
         expect(selectedEventsForFirstChildNode).toBe(firstChildNodeInTree.relatedEvents);
       });
-      it('should indicate the correct related event count for each category', () => {
-        const selectedRelatedInfo = selectors.relatedEventInfoByEntityId(store.getState());
-        const displayCountsForCategory = selectedRelatedInfo(firstChildNodeInTree.id)
-          ?.numberActuallyDisplayedForCategory!;
-
-        const eventCategoriesForNode: string[] = Object.keys(
-          eventStatsForFirstChildNode.byCategory
-        );
-
-        for (const eventCategory of eventCategoriesForNode) {
-          expect(`${eventCategory}:${displayCountsForCategory(eventCategory)}`).toBe(
-            `${eventCategory}:${eventStatsForFirstChildNode.byCategory[eventCategory]}`
-          );
-        }
-      });
-      /**
-       * The general approach reflected here is to _avoid_ showing a limit warning - even if we hit
-       * the overall related event limit - as long as the number in our category matches what the stats
-       * say we have. E.g. If the stats say you have 20 dns events, and we receive 20 dns events, we
-       * don't need to display a limit warning for that, even if we hit some overall event limit of e.g. 100
-       * while we were fetching the 20.
-       */
-      it('should not indicate the limit has been exceeded because the number of related events received for the category is greater or equal to the stats count', () => {
-        const selectedRelatedInfo = selectors.relatedEventInfoByEntityId(store.getState());
-        const shouldShowLimit = selectedRelatedInfo(firstChildNodeInTree.id)
-          ?.shouldShowLimitForCategory!;
-        for (const typeCounted of Object.keys(eventStatsForFirstChildNode.byCategory)) {
-          expect(shouldShowLimit(typeCounted)).toBe(false);
-        }
-      });
-      it('should not indicate that there are any related events missing because the number of related events received for the category is greater or equal to the stats count', () => {
-        const selectedRelatedInfo = selectors.relatedEventInfoByEntityId(store.getState());
-        const notDisplayed = selectedRelatedInfo(firstChildNodeInTree.id)
-          ?.numberNotDisplayedForCategory!;
-        for (const typeCounted of Object.keys(eventStatsForFirstChildNode.byCategory)) {
-          expect(notDisplayed(typeCounted)).toBe(0);
-        }
-      });
-      it('should return an overall correct count for the number of related events', () => {
-        const aggregateTotalByEntityId = selectors.relatedEventAggregateTotalByEntityId(
-          store.getState()
-        );
-        const countForId = aggregateTotalByEntityId(firstChildNodeInTree.id);
-        expect(countForId).toBe(aggregateCategoryTotalForFirstChildNode);
```

Comment:
:grey_question: I know we're not doing the aggregate count anymore but did we add tests anywhere to account for the "actual" count?

Reply:
We could write a test that confirms that the count coming form the API is represented in the UI (via simulator). We might have API tests for this as well. ES is doing the counting now. I should manually verify that the UI matches ES before merging.